Error when ig adding a crm-grid in an empty project.

### Description  
There's a `Cannot find module 'igniteui-angular-charts'` when ig adding a crm-grid

- Ignite UI CLI version: 5.0.0-rc.1
- Framework: Angular
- Project type: igx-ts

### Steps to reproduce  

Create a new project with `ng new newAngularProject --collection="@igniteui/angular-schematics" --template=side-nav`
2. Add a crm-grid: `ig add crm-grid newCrmGrid`
3. Start the project: `ng serve --open`

### Result  
A new window is opened but it just displays "Cannot GET /"
In the terminal console there's a 

`Cannot find module 'igniteui-angular-charts'.
11 import { IgxSparklineModule, IgxSparklineCoreModule } from 'igniteui-angular-charts';`

message.

--- a/packages/igx-templates/igx-ts/custom-templates/crm-grid/index.ts
+++ b/packages/igx-templates/igx-ts/custom-templates/crm-grid/index.ts
@@ -41,6 +41,7 @@ class IgxGridCRMTemplate extends IgniteUIForAngularTemplate {
 			},
 			{ import: "FormsModule", from: "@angular/forms" }
 		];
+		this.packages = ["tslib@^1.7.1", "igniteui-angular-core@~8.2.12", "igniteui-angular-charts@~8.2.12"];
 	}
 }
 module.exports = new IgxGridCRMTemplate();

